BridgeAPI.md

```diff
@@ -293,3 +296,115 @@ None
 
 ### Example
     QubitSDK.experienceShown("https://sse.qubit.com/v1/callback?data=igK....n0=");
+
+-------------------------------------------------------
+
+## **getPlacement**(placementId, mode, attributes, campaignId, experienceId, placementPromise)
```

Comment:
placementPromise is not a parameter of this Javascript function. Promise is a result.